### PR TITLE
re-named variables, new function for plot, docstrings

### PR DESCRIPTION
**Pull request by gdkwxn**
Added an if else function. If there is only one method, then the text displaying the method (e.g., 'X-RAY DIFFRACTION') will display horizontally rather than tilted 30 degrees vertically.  Else, if there is more than 1 method, then it prints how it always has.

Adjusted some variable names for clarification
readdir ---> read_directory_path

Added docstrings for description on what the functions do.

### CODE REVIEW

**Comment by fatipardo** [comment]
Is this the most recent pull request? Or should I wait?

**Comment by gdkwxn** [comment]
Hi, this is the most recent pull request. I was just trying to figure out if I needed to fix anything to the header or description to fit the requirements we put in the contributions file.

File: Notebooks/Analysis.SummaryPDBDataset.ipynb
Comment (fatipardo) [inline Notebooks/Analysis.SummaryPDBDataset.ipynb:1]: Line 62: data frame -> DataFrame
Line 82: armstrong -> angstrom
Line 208: change "organism" to output_name (or similar)

**Review by fatipardo — CHANGES_REQUESTED** [review]
Please make the following modifications:
Line 62: data frame -> DataFrame
Line 82: armstrong -> angstrom
Line 208: change "organism" to output_name (or similar)

**Review by fatipardo — APPROVED** [review]
All comments were addressed.Thank you!

**Pull request merged**